site/docs/operations/configuration.md

```diff
@@ -80,13 +80,64 @@ jetConfig.configureHazelcast(c -> {
 JetInstance jet = Jet.newJetInstance(jetConfig);
 ```
 
+### Configuration Loaders
+
 Alternatively, you can configure Jet to load its configuration from the
 classpath or working directory. By default it will search for
 `hazelcast-jet.yaml` and `hazelcast.yaml` files in the classpath and
 working directory, but you can control the name of the files using the
 relevant system properties, `hazelcast.jet.config` and
 `hazelcast.config`, respectively.
 
+All configuration loaders takes an optional `properties` parameter.
```

Comment:
Pls. use Grammarly, even the free version would immediately spot trivial grammar issues like "loaders takes".